### PR TITLE
Update Assignments GET APIs to include AppOnly support

### PR DESCRIPTION
**Pull request by zoexi**
Updating the EduAssignments docs to specify that we now support Applications Permissions for certain Assignments GET APIs (READ operations).

### CODE REVIEW

**Comment by Lauragra** [comment]
@zoexi , please follow the Get started steps to get set up as an internal contributor so you can follow our workflow process.

**Comment by zoexi** [comment]
> @zoexi , please follow the Get started steps to get set up as an internal contributor so you can follow our workflow process.

Is there another page I should reference? I was looking at this page earlier:
https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/CONTRIBUTING.md

Not sure how the Bot is tagging external/internal - via email addresses or smthg else. For example, this PR is tagged as external but prob should be tagged as internal as well: https://github.com/microsoftgraph/microsoft-graph-docs/pull/12365

**Comment by zoexi** [comment]
@Lauragra - can you pls advise here? I tried a couple more times - changed my git config user.email (before was using the Github provided private one) to my internal email address and my account is already linked.

**Comment by Lauragra** [comment]
> @Lauragra - can you pls advise here? I tried a couple more times - changed my git config user.email (before was using the Github provided private one) to my internal email address and my account is already linked.

Please follow the Get started steps in this topic to set yourself up as a contributor: https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/103184/Get-started-with-GitHub

**Comment by zoexi** [comment]
> > @Lauragra - can you pls advise here? I tried a couple more times - changed my git config user.email (before was using the Github provided private one) to my internal email address and my account is already linked.
> 
> Please follow the Get started steps in this topic to set yourself up as a contributor: https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/103184/Get-started-with-GitHub

Awesome, thanks! I think those are the steps I needed! It would be helpful it was linked somewhere in the CONTRIBUTING.md file - or mentioned somewhere for internal users. :)

**Review by dipakboyed — APPROVED** [review]
LGTM

**Comment by shravva** [comment]
Does this require to update changelog with change type: "Change" ? - @Lauragra

File: api-reference/beta/api/educationassignment-get-rubric.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ One of the following permissions is required to call this API. To learn more, in
 |:---------------------------------------|:--------------------------------------------|
 | Delegated (work or school account)     | EduAssignments.ReadBasic, EduAssignments.ReadWriteBasic, EduAssignments.Read, EduAssignments.ReadWrite |
 | Delegated (personal Microsoft account) | Not supported. |
-| Application                            | Not supported. |
+| Application                            | EduAssignments.ReadBasic, EduAssignments.ReadWriteBasic, EduAssignments.Read, EduAssignments.ReadWrite |
```
Comment (Lauragra) [inline api-reference/beta/api/educationassignment-get-rubric.md:26]: Do you need to add a changelog entry to call out that you're adding support for application scenarios?

Comment (zoexi) [reply api-reference/beta/api/educationassignment-get-rubric.md:26]: Can do!

**Comment by zoexi** [comment]
@Lauragra - I added the changelog entry. Pls let me know if anything else is required for this doc update, thank you for the review!

**Review by Lauragra — APPROVED** [review]

**Pull request merged**